packages/error-boundry-fallback/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@opensrp/error-boundry-fallback",
+  "version": "0.0.1",
+  "types": " dist/types",
```

Comment:
lets trim the space padding

Reply:
The file is indented and aligned, 
Do I have to trim something specifically ?